packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -56,6 +56,20 @@ import MQAProcessor from '../metrics/mqa-processor';
 
 import InMeetingActions from './in-meeting-actions';
 
+const logRequest = (request, {header = '', success = '', failure = ''}) => {
+  LoggerProxy.logger.log(header);
```

Comment:
Could you change these to `.info` instead of `.log`? I'm worried we are logging too much on default `log` setting.